community/pike/APKBUILD

```diff
@@ -0,0 +1,235 @@
+# Maintainer: Paweł Tomak <pawel@tomak.eu>
+pkgname=pike
+pkgver=8.0.610
+pkgrel=0
+pkgdesc="Pike Programing language"
+url="https://pike.lysator.liu.se"
+arch="x86_64"
+license="MIT"
+options="!check" # there is no tests for those packages
+depends=
+makedepends="gmp-dev \
+			bzip2-dev \
+			freeglut-dev \
+			gdbm-dev \
+			giflib-dev \
+			giflib-utils \
+			gtk+2.0 \
+			libgcrypt-dev \
+			libglade-dev \
+			libsm-dev \
+			mesa-gl \
+			nettle \
+			nettle-dev \
+			nettle-utils \
+			pcre-dev \
+			sdl-dev \
+			sdl_mixer-dev \
+			sqlite-dev \
+			sqlite-libs \
+			readline \
+			zlib-dev \
+			librsvg-dev"
+install=
+subpackages="$pkgname-bzip2 \
+			$pkgname-dnssd \
+			$pkgname-fuse \
+			$pkgname-gdbm \
+			$pkgname-gl \
+			$pkgname-gtk \
+			$pkgname-image \
+			$pkgname-pcre \
+			$pkgname-sdl \
+			$pkgname-svg \
+			$pkgname-sqlite \
+			$pkgname-dev \
+			$pkgname-doc \
+			$pkgname-core"
+source="http://pike.lysator.liu.se/pub/pike/all/${pkgver}/Pike-v${pkgver}.tar.gz"
+builddir="$srcdir/"
+
+unpack() {
+	cd "$builddir"
+	tar xzf Pike-v${pkgver}.tar.gz --strip-components=1
+}
+
+build() {
+	cd "$builddir"
+	make CONFIGUREARGS=" \
+		--prefix=/usr \
+		--disable-make_conf \
+		--disable-noopty-retry \
+		--without-cdebug \
+		--without-bundles \
+		--without-ssleay \
+		--with-gmp \
+		--with-crypt \
+		--with-bignums \
+		--without-rtldebug \
+		--with-Bz2 \
+		--without-fftw \
+		--without-_Ffmpeg \
+		--with-gdbm \
+		--without-GTK1 \
+		--with-GTK2 \
+		--with-jpeglib \
+		--without-Kerberos \
+		--without-msql \
+		--without-mysql \
+		--without-Odbc \
+		--without-oracle \
+		--with-GL \
+		--with-GLUT \
+		--with-_Regexp_PCRE \
+		--without-libpdf \
+		--without-sass \
+		--with-sane \
+		--with-SDL \
+		--with-SDL_Mixer \
+		--with-svg \
+		--with-tifflib \
+		--without-freetype \
+		--without-vcdiff \
+		--with-_Image_WebP \
+		--with-zlib"
+}
+
+package() {
+	cd "$builddir"
+	depends=""
```

Comment:
Do you have a reason to reset `depends` from the top of the file here? I'd be confused if I added an item to `depends` as I normally would at the top and it didn't end up in the built package.

Reply:
Probably some kind of leftover from development